src/opencmiss/utils/zinc/finiteelement.py

```diff
@@ -172,41 +172,41 @@ def get_node_name_centres(nodeset: Nodeset, coordinatesField: Field, nameField:
     return nameCentres
 
 
-def evaluate_nodeset_coordinates_range(coordinates: Field, nodeset: Nodeset):
+def evaluate_field_nodeset_range(field: Field, nodeset: Nodeset):
     """
-    :return: min, max range of coordinates field over nodes.
+    :return: min, max range of field over nodes.
     """
     fieldmodule = nodeset.getFieldmodule()
-    componentsCount = coordinates.getNumberOfComponents()
-    with ZincCacheChanges(fieldmodule):
-        minCoordinates = fieldmodule.createFieldNodesetMinimum(coordinates, nodeset)
-        maxCoordinates = fieldmodule.createFieldNodesetMaximum(coordinates, nodeset)
+    components_count = field.getNumberOfComponents()
+    with ChangeManager(fieldmodule):
+        min_field = fieldmodule.createFieldNodesetMinimum(field, nodeset)
+        max_field = fieldmodule.createFieldNodesetMaximum(field, nodeset)
         fieldcache = fieldmodule.createFieldcache()
-        result, minX = minCoordinates.evaluateReal(fieldcache, componentsCount)
+        result, min_values = min_field.evaluateReal(fieldcache, components_count)
         assert result == RESULT_OK
-        result, maxX = maxCoordinates.evaluateReal(fieldcache, componentsCount)
+        result, max_values = max_field.evaluateReal(fieldcache, components_count)
         assert result == RESULT_OK
-        del minCoordinates
-        del maxCoordinates
+        del min_field
+        del max_field
         del fieldcache
-    return minX, maxX
+    return min_values, max_values
 
 
-def evaluate_nodeset_mean_coordinates(coordinates: Field, nodeset: Nodeset):
+def evaluate_field_nodeset_mean(field: Field, nodeset: Nodeset):
     """
-    :return: Mean of coordinates over nodeset.
+    :return: Mean of field over nodeset.
     """
     fieldmodule = nodeset.getFieldmodule()
-    componentsCount = coordinates.getNumberOfComponents()
-    with ZincCacheChanges(fieldmodule):
-        meanCoordinatesField = fieldmodule.createFieldNodesetMean(coordinates, nodeset)
+    components_count = field.getNumberOfComponents()
+    with ChangeManager(fieldmodule):
+        mean_field = fieldmodule.createFieldNodesetMean(field, nodeset)
         fieldcache = fieldmodule.createFieldcache()
-        result, meanCoordinates = meanCoordinatesField.evaluateReal(fieldcache, componentsCount)
+        result, mean_values = mean_field.evaluateReal(fieldcache, components_count)
         assert result == RESULT_OK
-        del meanCoordinatesField
+        del mean_field
         del fieldcache
     assert result == RESULT_OK
```

Comment:
I don't think this second assert is required.

Reply:
Fixed

Reply:
Fixed.